packages/fluent/scss/checkbox/_layout.scss

```diff
@@ -255,8 +255,7 @@
             top: 50%;
             border-radius: 100%;
             z-index: -1;
-            transition: opacity 100ms linear, transform 150ms cubic-bezier(.4, 0, .2, 1);
-            transform: translate(-50%, -50%) scale(0);
+            transition: k-motion(short-2, linear, opacity), k-motion(short-3, ease-in-out, transform);
             transform-origin: center center;
```

Comment:
Is `transform: translate(-50%, -50%) scale(0);` deleted on purpose?

Reply:
No, it shouldn't have been deleted